doc/man1/flux-account-add-queue.rst

```diff
@@ -0,0 +1,40 @@
+.. flux-help-section: flux account
+
+=========================
+flux-account-add-queue(1)
+=========================
+
+
+SYNOPSIS
+========
+
+**flux** **account** **add-queue** QUEUE [OPTIONS]
+
+DESCRIPTION
+===========
+
+.. program:: flux account add-queue
+
+:program:`flux account add-queue` will add a queue to the ``queue_table`` in
+the flux-accounting database. Different properties and limits can be set for
```

Comment:
Somewhere (but I'm having trouble thinking of where it makes sense) I would like to see something along the lines of 

> Note that flux-accounting queues are for job accounting purposes, and a separate, disparate construct from [queues in flux-core](https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man5/flux-config-queues.html) and [queue policies in flux-sched](https://flux-framework.readthedocs.io/projects/flux-sched/en/latest/man5/flux-config-sched-fluxion-qmanager.html).

Reply:
That's a good thought! Maybe I should add it as a note at the top of these man `*-queue` pages.

Reply:
In my note above, s/queue policies in flux-sched/_queuing_ policies in flux-sched/